miio/heater_miot.py

```diff
@@ -0,0 +1,209 @@
+import enum
+import logging
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .exceptions import DeviceException
+from .miot_device import MiotDevice
+
+_LOGGER = logging.getLogger(__name__)
+_MAPPING = {
+    # Source https://miot-spec.org/miot-spec-v2/instance\?type\=urn:miot-spec-v2:device:heater:0000A01A:zhimi-mc2:1
+    # Heater (siid=2)
+    "power": {"siid": 2, "piid": 1},
+    "target_temperature": {"siid": 2, "piid": 5},
+    # Countdown (siid=3)
+    "countdown_time": {"siid": 3, "piid": 1},
+    # Environment (siid=4)
+    "temperature": {"siid": 4, "piid": 7},
+    # Physical Control Locked (siid=6)
+    "child_lock": {"siid": 5, "piid": 1},
+    # Alarm (siid=6)
+    "buzzer": {"siid": 6, "piid": 1},
+    # Indicator light (siid=7)
+    "led_brightness": {"siid": 7, "piid": 3},
+}
+
+
+class LedBrightness(enum.Enum):
+    On = 0
+    Off = 1
+
+
+class HeaterMiotException(DeviceException):
+    pass
+
+
+class HeaterMiotStatus:
+    """Container for status reports from the heater which uses the MIoT protocol."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
```

Comment:
Please add an example data dictionary to the docstring, see other miot devices for example.

Reply:
Also, please update the README.md to add this to supported devices.